definitions/features/downstream.rb

```diff
@@ -74,6 +74,14 @@ def rh_repos(sat_version)
     rh_repos
   end
 
+  def enable_ansible_repo(sat_version, rh_repos, rh_version_major)
+    if [version('6.4'), version('6.5')].include?(sat_version)
```

Comment:
This condition is going to fail for z-stream versions i.e `6.4.z & 6.5.z`

Could you please modify this condition to `sat_version >= version('6.4') && sat_version < version('6.6')` or I would suggest to modify if..else block similar to 

```

if sat_version >= version('6.6')
   rh_repos << "rhel-#{rh_version_major}-server-ansible-2.8-rpms"
elsif sat_version >= version('6.4')
   rh_repos << "rhel-#{rh_version_major}-server-ansible-2.6-rpms"
end
```

Reply:
@kgaikwad Thank you for help, changed this and its working perfectly!

Reply:
@upadhyeammit, 
I would except that new method should return appropriate ansible_repo name instead of modifying `rh_repos` inside new method. 
For example -
`rh_repos <<  {your_new_method_call(sat_version)} if sat_version > version('6.3')`